identity/handler.go

```diff
@@ -163,6 +172,40 @@ func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 		return
 	}
 
+	identifier := r.URL.Query().Get("identifier")
```

Comment:
Please add a test and documentation for this functionality.

Also, it appears that this code is doing some extra work - we already have access to the credentials (by calling `h.r.PrivilegedIdentityPool().GetIdentityConfidential`) so there is no need to look them up again.

I am also wondering if we could have a better naming pattern for this? Looking at it without context,

```
GET /identities/<id>?identifier=something
```

isn't particularly revealing in terms of what we would expect. So should this be its own endpoint, or maybe a query parameter that more clearly states what we are doing here? For example `?reveal_credentials=oidc_tokens` or something similar.

Reply:
You right, it's sound good to me

Reply:
So that means it will show all provider oidc